### PR TITLE
(DOCSP-14677): Node.js Local Quick Start

### PR DESCRIPTION
**Pull request by mongomoe**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-14677

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-14677-Nodejs-Local-QuickStart/sdk/node/quick-start-local/

### Docs staging build link:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=6038622d75e817aef7217242

### CODE REVIEW

File: source/sdk/node/quick-start-local.txt [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    
+   .. literalinclude:: /examples/generated/node/quickstart-local.codeblock.quickstart-local-define-an-object-model.js
+     :language: javascript
+
+.. _node-quickstart-local-open-a-realm:
+
+
+Open a Realm
+------------
+To open a local (non-synced) {+realm+}, pass a :js-sdk:`Realm.Configuration
```
Comment (cbush) [inline source/sdk/node/quick-start-local.txt:78]: I think we should think of "local-only" as just default, as our readers likely will too.

```suggestion
To open a {+realm+}, pass a :js-sdk:`Realm.Configuration
```

**Review by cbush — APPROVED** [review]

File: source/sdk/node/quick-start-local.txt (hunk 1 above)
Comment (mongomoe) [reply source/sdk/node/quick-start-local.txt]: 👍

File: source/sdk/node/quick-start-local.txt [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+   .. literalinclude:: /examples/generated/node/quickstart-local.codeblock.delete-an-object.js
+     :language: javascript
+
+.. _node-quickstart-local-watch-a-collection:
+
+Watch a Collection
+------------------
+You can :ref:`watch a realm, collection, or object for changes
+<node-change-notifications>` by registering event handlers with the
+:js-sdk:`Realm.addListener() <Realm.html#addListener>`
```
Comment (cbush) [inline source/sdk/node/quick-start-local.txt:141]: These links seems to be broken on staging?

**Review by cbush — APPROVED** [review]
Looks great, please check those links to confirm.

File: source/sdk/node/quick-start-local.txt (hunk 2 above)
Comment (mongomoe) [reply source/sdk/node/quick-start-local.txt:141]: Hmm, this is odd. The links are right and should link out to "https://docs.mongodb.com/realm-sdks/js/latest/<TypeThatYouAreWatchingForChangesOn>.html#addListener"  but instead link out the same thing but with a "/" before the "#addListener" (so "https://docs.mongodb.com/realm-sdks/js/latest/Realm.html/#addListener") which leads to a 404.

Possibly a platform issue/change?

Comment (mongomoe) [reply source/sdk/node/quick-start-local.txt:141]: Looks like nate already reported the issue to platform - https://mongodb.slack.com/archives/C018UTGP23T/p1614375275100300

Comment (mongomoe) [reply source/sdk/node/quick-start-local.txt:141]: Merging because this is a platform issue rather than an issue with this specific pr

**Pull request merged**